### ISSUE TITLE
Find a better way to stay off of "bad reputation" site lists

### ISSUE THREAD

**Issue by pi-rho** [dirtbags/moth#64]
https://github.com/dirtbags/moth/blob/a3636c0a5566d815083431f830fa456331d87696/theme/moth.js#L153-L161

That `if` ...

**Comment by pi-rho** [comment]
https://github.com/dirtbags/moth/blob/a3636c0a5566d815083431f830fa456331d87696/theme/moth.js#L156

**Comment by pi-rho** [comment]
https://github.com/dirtbags/moth/blob/a3636c0a5566d815083431f830fa456331d87696/theme/moth.js#L103-L113

And `showPuzzles(teamId)` is writing teamId to sessionStorage, which is wrong. This function should be showing puzzles, not "logging in" a user.

**Comment by pi-rho** [comment]
https://github.com/dirtbags/moth/blob/a3636c0a5566d815083431f830fa456331d87696/src/handlers.go#L170-L180

And here we see the teamId is just a fence for the data. This is wrong.

**Comment by nealey** [comment]
The current code was added at the suggestion of the Cyber Fire staff, after a pretty long discussion, as a "hail mary" attempt to get our puzzle server off of a "bad reputation" list. The thinking was that some spider crawled the site, found malware, and essentially blocked us; by removing the ability to crawl puzzles without a token, we removed this avenue to discovery of puzzles by spiders.

We don't know if this is going to help, but it's the best we could come up with. The right solution would be:

1. Find out how we got on this list
1. Find out how we can get off this list
1. Find out how we, and anyone else running MOTH, can stay off this list

Until we can figure this out (and just getting the first one seems impossible, as the criteria appears to be company proprietary), we're going to have the least intrusive solution to our best collective educated guess, which was to require a team ID to list open puzzles.
